### ISSUE TITLE
C:\Users\username\AppData\Local\sfdx\client\bin\..\7.110.0-85b006b\bin\sfdx.cmd"' is not recognized as an internal or external command, operable program or batch file.

### ISSUE THREAD

**Issue by CXCAlfo** [forcedotcom/salesforcedx-vscode#3471]
### Summary

A number of my developers (3 - 2 in India team and 1 in Melbourne team) started getting the SFDX Command not recognised issue yesterday evening.  None of them have run sfdx update. 

"C:\Users\username\AppData\Local\sfdx\client\bin\..\7.110.0-85b006b\bin\sfdx.cmd"' is not recognized as an internal or external command, operable program or batch file.

### Steps To Reproduce:

Run any SFDX Comment in terminal window.

### Expected result

Command runs successfully

### Actual result

"C:\Users\username\AppData\Local\sfdx\client\bin\..\7.110.0-85b006b\bin\sfdx.cmd"' is not recognized as an internal or external command, operable program or batch file.

### Additional information
 I can't get them to post Plugin versions because SFDX Commands wont run.

**Comment by yippie** [comment]
This happened to me and one of my co-workers last night also. We are on macOS.

Our work around was to delete the entire Local\sfdx folder then re-install the sfdx CLI. Just reinstalling sfdx did not work until I also deleted the folder.

Some scheduled script or plugin update just hosed SFDX for existing installs it seems

**Comment by yippie** [comment]
Probably should cross reference the CLI issue https://github.com/forcedotcom/cli/issues/1104

**Comment by smaddox-sf** [comment]
Thanks for reaching out on this and for relating it to the other open issue.  We are tracking this issue as the CLI issue @yippie correctly linked.  Please check out this comment for status info and a workaround - 
https://github.com/forcedotcom/cli/issues/1104#issuecomment-892827077

Closing this issue to consolidate.  Please follow forcedotcom/cli#1104 for the latest info on this issue

**Comment by suniti-majumdar** [comment]
I was facing this error whenever I tried to run a simple sfdx command in my VS code terminal

"C:\Users\XXXXXXX\AppData\Local\sfdx\client\bin\..\7.209.6-8ba3197\bin\sfdx.cmd"' is not recognized as an internal or external command,
operable program or batch file.
I found the below issues, and fix for it. It worked.

1. I had 2 sfdx folders one in the C:\Program Files\sfdx folder and the other in C:\Users\XXXXXXX\AppData\Local\sfdx.
2. I uninstalled sfdx-cli from the control panel and deleted sfdx folder from the above paths
3. Then install sfdxcli from here https://developer.salesforce.com/tools/salesforcecli
4.  set the user variable and system variable for it   as C:\Program Files\sf
5. Then restart vs code and run sf update command in the terminal window.
